core/lib/test_contracts/contracts/simple-transfer/simple-transfer.sol

```diff
@@ -19,13 +19,15 @@ contract SimpleTransfer {
     // Function to withdraw Ether to the owner's address
     function withdraw(uint _amount) public onlyOwner {
         require(address(this).balance >= _amount, "Insufficient balance in contract");
-        payable(owner).transfer(_amount);
+        (bool success, ) = owner.call{value: _amount}("");
+        require(success, "transfer reverted");
```

Comment:
Not a Solidity expert: is this change OK? AFAICT, these methods aren't currently used in `SimpleTransfer` anyway. It's used only in [a single test for the prestate tracer](https://github.com/matter-labs/zksync-era/blob/474bc04e9a2e4ab03b261b55fda6cc37a27961f8/core/lib/multivm/src/versions/vm_latest/tests/prestate_tracer.rs#L62), but I had no luck so far replacing it with another contract. AFAIU, it matters that the contract is stateful, its state is written to in the constructor, and it accepts payments. Generally: is the prestate tracer used anywhere? It doesn't look used in the Era repo.